api/apps/api/test/scenarios/input-file.e2e-spec.ts

```diff
@@ -203,6 +230,16 @@ async function getFixtures() {
         .set('Authorization', `Bearer ${jwtToken}`)
         .send(input);
     },
+    async ThenScenarioHasMetadataForMarxan(id: string) {
```

Comment:
Will probably move later on to https://github.com/Vizzuality/marxan-cloud/pull/306